Test failure: COR_E_FAILFAST while running System.IO.FileSystem.Tests

Ran on Surface Pro 4, Korean Windows 10 Pro 1809 (Build 17763.316)

```
C:\Users\gotos\.nuget\packages\microsoft.dotnet.corefxtesting\1.0.0-beta.19126.3\build\core\Core.targets(190,5): error MSB3073: The command "C:\Users\gotos\source\repos\corefx\artifacts\bin\tests\System.IO.FileSystem.Tests\netcoreapp-Windows_NT-Debug-x64\RunTests.cmd C:\Users\gotos\source\repos\corefx\artifacts\bin\testhost/netcoreapp-Windows_NT-Debug-x64/ C:\Users\gotos\source\repos\corefx\.dotnet\ C:\Users\gotos\source\repos\corefx\artifacts\tools\" exited with code -2146232797. [C:\Users\gotos\source\repos\corefx\src\System.IO.FileSystem\tests\System.IO.FileSystem.Tests.csproj]
C:\Users\gotos\.nuget\packages\microsoft.dotnet.corefxtesting\1.0.0-beta.19126.3\build\core\Core.targets(201,5): error : One or more tests failed while running tests from 'System.IO.FileSystem.Tests'. [C:\Users\gotos\source\repos\corefx\src\System.IO.FileSystem\tests\System.IO.FileSystem.Tests.csproj]
```
when executing the given command separately in cmd, I got this:

```
C:\Users\gotos\source\repos\corefx\artifacts\bin\tests\System.IO.FileSystem.Tests\netcoreapp-Windows_NT-Debug-x64>C:\Users\gotos\source\repos\corefx\artifacts\bin\testhost\netcoreapp-Windows_NT-Debug-x64\dotnet.exe xunit.console.dll System.IO.FileSystem.Tests.dll -xml testResults.xml -nologo -notrait category=nonnetcoreapptests -notrait category=nonwindowstests -notrait category=failing -notrait category=OuterLoop
  Discovering: System.IO.FileSystem.Tests (method display = ClassAndMethod, method display options = None)
이 작업을 수행할 수 있는 권한이 없습니다. //Translation: You do not have sufficient privilege to perform this operation.
  Discovered:  System.IO.FileSystem.Tests (found 2934 of 3513 test cases)
  Starting:    System.IO.FileSystem.Tests (parallel test collections = on, max threads = 4)
    System.IO.Tests.DirectoryInfo_Delete_bool.DeletingSymLinkDoesntDeleteTarget [SKIP]
      Condition(s) not met: "CanCreateSymbolicLinks"
    System.IO.Tests.DirectoryInfo_EnumerateFiles_Str.EnumerateWithSymLinkToFile [SKIP]
      Condition(s) not met: "CanCreateSymbolicLinks"
    System.IO.Tests.FileInfo_Exists.SymLinksMayExistIndependentlyOfTarget [SKIP]
      Condition(s) not met: "CanCreateSymbolicLinks"
    System.IO.Tests.File_GetSetTimes.TimesIncludeMillisecondPart_HFS [SKIP]
      Condition(s) not met: "isHFS"
    System.IO.Tests.DirectoryInfo_Create.RootPath_AppContainer [SKIP]
      Condition(s) not met: "IsInAppContainer"
    System.IO.Tests.File_Exists.SymLinksMayExistIndependentlyOfTarget [SKIP]
      Condition(s) not met: "CanCreateSymbolicLinks"
    System.IO.Tests.DirectoryInfo_GetFiles.EnumerateWithSymLinkToFile [SKIP]
      Condition(s) not met: "CanCreateSymbolicLinks"
Assertion Failed
Unexpected error code getting attributes 121

   at System.IO.FileSystem.FillAttributeInfo(String path, WIN32_FILE_ATTRIBUTE_DATA& data, Boolean returnErrorOnNotFound) in C:\Users\gotos\source\repos\corefx\src\System.IO.FileSystem\src\System\IO\FileSystem.Windows.cs:line 224
   at System.IO.FileSystem.DirectoryExists(String path, Int32& lastError) in C:\Users\gotos\source\repos\corefx\src\System.IO.FileSystem\src\System\IO\FileSystem.Windows.cs:line 187
   at System.IO.FileSystem.DirectoryExists(String fullPath) in C:\Users\gotos\source\repos\corefx\src\System.IO.FileSystem\src\System\IO\FileSystem.Windows.cs:line 181
   at System.IO.FileSystem.CreateDirectory(String fullPath) in C:\Users\gotos\source\repos\corefx\src\System.IO.FileSystem\src\System\IO\FileSystem.Windows.cs:line 70
   at System.IO.Directory.CreateDirectory(String path) in C:\Users\gotos\source\repos\corefx\src\System.IO.FileSystem\src\System\IO\Directory.cs:line 47
   at System.IO.Tests.Directory_CreateDirectory.Create(String path) in C:\Users\gotos\source\repos\corefx\src\System.IO.FileSystem\tests\Directory\CreateDirectory.cs:line 17
   at System.IO.Tests.Directory_CreateDirectory.<>c__DisplayClass44_0.<PathWithReservedDeviceNameAsPath_ThrowsDirectoryNotFoundException>b__0() in C:\Users\gotos\source\repos\corefx\src\System.IO.FileSystem\tests\Directory\CreateDirectory.cs:line 517
   at Xunit.Assert.RecordException(Func`1 testCode) in C:\projects\xunit\src\xunit.assert\Asserts\Record.cs:line 50
   at Xunit.Assert.Throws[T](Func`1 testCode) in C:\projects\xunit\src\xunit.assert\Asserts\ExceptionAsserts.cs:line 41
   at System.IO.Tests.Directory_CreateDirectory.PathWithReservedDeviceNameAsPath_ThrowsDirectoryNotFoundException(String path) in C:\Users\gotos\source\repos\corefx\src\System.IO.FileSystem\tests\Directory\CreateDirectory.cs:line 517
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture) in F:\vsagent\19\s\src\System.Private.CoreLib\src\System\Reflection\RuntimeMethodInfo.cs:line 466
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Reflection\MethodBase.cs:line 54
   at Xunit.Sdk.TestInvoker`1.CallTestMethod(Object testClassInstance) in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestInvoker.cs:line 150
   at Xunit.Sdk.TestInvoker`1.<>c__DisplayClass48_1.<<InvokeTestMethodAsync>b__1>d.MoveNext() in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestInvoker.cs:line 257
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 226
   at Xunit.Sdk.TestInvoker`1.<>c__DisplayClass48_1.<InvokeTestMethodAsync>b__1()
   at Xunit.Sdk.ExecutionTimer.AggregateAsync(Func`1 asyncAction) in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\ExecutionTimer.cs:line 48
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 226
   at Xunit.Sdk.ExecutionTimer.AggregateAsync(Func`1 asyncAction)
   at Xunit.Sdk.TestInvoker`1.<>c__DisplayClass48_1.<InvokeTestMethodAsync>b__0() in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestInvoker.cs:line 242
   at Xunit.Sdk.ExceptionAggregator.RunAsync(Func`1 code) in C:\projects\xunit\src\xunit.core\Sdk\ExceptionAggregator.cs:line 90
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 226
   at Xunit.Sdk.ExceptionAggregator.RunAsync(Func`1 code)
   at Xunit.Sdk.TestInvoker`1.InvokeTestMethodAsync(Object testClassInstance) in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestInvoker.cs:line 241
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 353
   at Xunit.Sdk.TestInvoker`1.InvokeTestMethodAsync(Object testClassInstance)
   at Xunit.Sdk.XunitTestInvoker.InvokeTestMethodAsync(Object testClassInstance) in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\XunitTestInvoker.cs:line 112
   at Xunit.Sdk.TestInvoker`1.<RunAsync>b__47_0() in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestInvoker.cs:line 206
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 353
   at Xunit.Sdk.TestInvoker`1.<RunAsync>b__47_0()
   at Xunit.Sdk.ExceptionAggregator.RunAsync[T](Func`1 code) in C:\projects\xunit\src\xunit.core\Sdk\ExceptionAggregator.cs:line 107
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 353
   at Xunit.Sdk.ExceptionAggregator.RunAsync[T](Func`1 code)
   at Xunit.Sdk.TestInvoker`1.RunAsync() in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestInvoker.cs:line 189
   at Xunit.Sdk.XunitTestRunner.InvokeTestMethodAsync(ExceptionAggregator aggregator) in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\XunitTestRunner.cs:line 84
   at Xunit.Sdk.XunitTestRunner.InvokeTestAsync(ExceptionAggregator aggregator) in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\XunitTestRunner.cs:line 67
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 353
   at Xunit.Sdk.XunitTestRunner.InvokeTestAsync(ExceptionAggregator aggregator)
   at Xunit.Sdk.TestRunner`1.<>c__DisplayClass43_0.<RunAsync>b__0() in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestRunner.cs:line 149
   at Xunit.Sdk.ExceptionAggregator.RunAsync[T](Func`1 code) in C:\projects\xunit\src\xunit.core\Sdk\ExceptionAggregator.cs:line 107
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 353
   at Xunit.Sdk.ExceptionAggregator.RunAsync[T](Func`1 code)
   at Xunit.Sdk.TestRunner`1.RunAsync() in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestRunner.cs:line 149
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 353
   at Xunit.Sdk.TestRunner`1.RunAsync()
   at Xunit.Sdk.XunitTheoryTestCaseRunner.RunTestAsync() in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\XunitTheoryTestCaseRunner.cs:line 144
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 353
   at Xunit.Sdk.XunitTheoryTestCaseRunner.RunTestAsync()
   at Xunit.Sdk.TestCaseRunner`1.RunAsync() in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestCaseRunner.cs:line 82
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 353
   at Xunit.Sdk.TestCaseRunner`1.RunAsync()
   at Xunit.Sdk.XunitTheoryTestCase.RunAsync(IMessageSink diagnosticMessageSink, IMessageBus messageBus, Object[] constructorArguments, ExceptionAggregator aggregator, CancellationTokenSource cancellationTokenSource) in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\XunitTheoryTestCase.cs:line 51
   at Xunit.Sdk.XunitTestMethodRunner.RunTestCaseAsync(IXunitTestCase testCase) in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\XunitTestMethodRunner.cs:line 45
   at Xunit.Sdk.TestMethodRunner`1.RunTestCasesAsync() in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestMethodRunner.cs:line 136
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 353
   at Xunit.Sdk.TestMethodRunner`1.RunTestCasesAsync()
   at Xunit.Sdk.TestMethodRunner`1.RunAsync() in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestMethodRunner.cs:line 106
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 353
   at Xunit.Sdk.TestMethodRunner`1.RunAsync()
   at Xunit.Sdk.XunitTestClassRunner.RunTestMethodAsync(ITestMethod testMethod, IReflectionMethodInfo method, IEnumerable`1 testCases, Object[] constructorArguments) in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\XunitTestClassRunner.cs:line 168
   at Xunit.Sdk.TestClassRunner`1.RunTestMethodsAsync() in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestClassRunner.cs:line 213
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 353
   at Xunit.Sdk.TestClassRunner`1.RunTestMethodsAsync()
   at Xunit.Sdk.TestClassRunner`1.RunAsync() in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestClassRunner.cs:line 171
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 353
   at Xunit.Sdk.TestClassRunner`1.RunAsync()
   at Xunit.Sdk.XunitTestCollectionRunner.RunTestClassAsync(ITestClass testClass, IReflectionTypeInfo class, IEnumerable`1 testCases) in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\XunitTestCollectionRunner.cs:line 158
   at Xunit.Sdk.TestCollectionRunner`1.RunTestClassesAsync() in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestCollectionRunner.cs:line 130
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 353
   at Xunit.Sdk.TestCollectionRunner`1.RunTestClassesAsync()
   at Xunit.Sdk.TestCollectionRunner`1.RunAsync() in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestCollectionRunner.cs:line 101
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 1028
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Runtime\CompilerServices\AsyncMethodBuilder.cs:line 353
   at Xunit.Sdk.TestCollectionRunner`1.RunAsync()
   at Xunit.Sdk.XunitTestAssemblyRunner.RunTestCollectionAsync(IMessageBus messageBus, ITestCollection testCollection, IEnumerable`1 testCases, CancellationTokenSource cancellationTokenSource) in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\XunitTestAssemblyRunner.cs:line 235
   at Xunit.Sdk.XunitTestAssemblyRunner.<>c__DisplayClass14_2.<RunTestCollectionsAsync>b__2() in C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\XunitTestAssemblyRunner.cs:line 184
   at System.Threading.Tasks.Task`1.InnerInvoke() in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Threading\Tasks\Future.cs:line 517
   at System.Threading.Tasks.Task.<>c.<.cctor>b__277_0(Object obj) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Threading\Tasks\Task.cs:line 2463
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Threading\ExecutionContext.cs:line 173
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Threading\Tasks\Task.cs:line 2420
   at System.Threading.Tasks.Task.ExecuteEntry() in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Threading\Tasks\Task.cs:line 2331
   at System.Threading.Tasks.SynchronizationContextTaskScheduler.<>c.<.cctor>b__8_0(Object s) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Threading\Tasks\TaskScheduler.cs:line 652
   at Xunit.Sdk.MaxConcurrencySyncContext.RunOnSyncContext(SendOrPostCallback callback, Object state) in C:\projects\xunit\src\xunit.execution\Sdk\MaxConcurrencySyncContext.cs:line 107
   at Xunit.Sdk.MaxConcurrencySyncContext.<>c__DisplayClass11_0.<WorkerThreadProc>b__0(Object _) in C:\projects\xunit\src\xunit.execution\Sdk\MaxConcurrencySyncContext.cs:line 96
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Threading\ExecutionContext.cs:line 173
   at Xunit.Sdk.ExecutionContextHelper.Run(Object context, Action`1 action) in C:\projects\xunit\src\xunit.execution\Sdk\Utility\ExecutionContextHelper.cs:line 111
   at Xunit.Sdk.MaxConcurrencySyncContext.WorkerThreadProc() in C:\projects\xunit\src\xunit.execution\Sdk\MaxConcurrencySyncContext.cs:line 89
   at Xunit.Sdk.XunitWorkerThread.<>c.<.ctor>b__1_0(Object s) in C:\projects\xunit\src\common\XunitWorkerThread.cs:line 72
   at System.Threading.Thread.ThreadMain_ParameterizedThreadStart(Object parameter) in C:\Users\gotos\source\repos\corefx\src\System.Threading.Thread\src\System\Threading\Thread.cs:line 116
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state) in F:\vsagent\19\s\src\System.Private.CoreLib\src\System\Threading\Thread.cs:line 54
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) in F:\vsagent\19\s\src\System.Private.CoreLib\shared\System\Threading\ExecutionContext.cs:line 173
   at System.Threading.ThreadHelper.ThreadStart(Object obj) in F:\vsagent\19\s\src\System.Private.CoreLib\src\System\Threading\Thread.cs:line 69
----- end 21:32:09.80 ----- exit code -2146232797 ----------------------------------------------------------
```

--- a/src/libraries/Common/src/System/IO/FileSystem.Attributes.Windows.cs
+++ b/src/libraries/Common/src/System/IO/FileSystem.Attributes.Windows.cs
@@ -81,7 +81,7 @@ namespace System.IO
                         )
                     {
                         // Assert so we can track down other cases (if any) to add to our test suite
-                        Debug.Assert(errorCode == Interop.Errors.ERROR_ACCESS_DENIED || errorCode == Interop.Errors.ERROR_SHARING_VIOLATION,
+                        Debug.Assert(errorCode == Interop.Errors.ERROR_ACCESS_DENIED || errorCode == Interop.Errors.ERROR_SHARING_VIOLATION || Interop.Errors.ERROR_SEM_TIMEOUT,
                             $"Unexpected error code getting attributes {errorCode} from path {path}");
 
                         // Files that are marked for deletion will not let you GetFileAttributes,

--- a/src/libraries/Common/src/System/IO/FileSystem.Attributes.Windows.cs
+++ b/src/libraries/Common/src/System/IO/FileSystem.Attributes.Windows.cs
@@ -81,7 +81,7 @@ namespace System.IO
                         )
                     {
                         // Assert so we can track down other cases (if any) to add to our test suite
-                        Debug.Assert(errorCode == Interop.Errors.ERROR_ACCESS_DENIED || errorCode == Interop.Errors.ERROR_SHARING_VIOLATION || Interop.Errors.ERROR_SEM_TIMEOUT,
+                        Debug.Assert(errorCode == Interop.Errors.ERROR_ACCESS_DENIED || errorCode == Interop.Errors.ERROR_SHARING_VIOLATION || errorCode == Interop.Errors.ERROR_SEM_TIMEOUT,
                             $"Unexpected error code getting attributes {errorCode} from path {path}");
 
                         // Files that are marked for deletion will not let you GetFileAttributes,

